### DISCUSSION TITLE
List of Lifetime

### DISCUSSION THREAD

**Discussion by giscus** [edony-ink/comments#21] [General]
# List of Lifetime

Something I like to do of my whole lifetime.

https://www.edony.ink/private/list-of-lifetime/

**Comment by edonyzpc** [comment]
NOTE: 清单列表依托 gist 做自动托管，所以因为众所周知的原因，墙内无法完整刷新出网页。

**Comment by FX-Max** [comment]
加油，期待都打上勾的那天。

**Reply by edonyzpc** [reply]
谢谢，还在努力中😊
